Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM behance/docker-base:2.6-ubuntu-18.04
 MAINTAINER Bryan Latten <latten@adobe.com>
 
+# Use in multi-phase builds, when an init process requests for the container to gracefully exit, so that it may be committed
+# Used with alternative CMD (worker.sh), leverages supervisor to maintain long-running processes
 ENV CONTAINER_ROLE=web \
     CONTAINER_PORT=8080 \
     CONF_NGINX_SITE="/etc/nginx/sites-available/default" \
     CONF_NGINX_SERVER="/etc/nginx/nginx.conf" \
-    NOT_ROOT_USER=www-data
+    NOT_ROOT_USER=www-data \
+    S6_KILL_FINISH_MAXTIME=55000
```

Comment:
@bryanlatten : One thing we found out is that this value might be limited to less than 65.5 seconds (64k effectively) on Alpine: https://jira.corp.adobe.com/browse/EON-4242. Would you think its worthwhile to link to the JIRA and our the resources I link off it?

I guess if not here, maybe in the README. Also maybe just a note as to why 55 seconds was chosen (even if it was arbitrary) so that we'll remember why we did it. (We have a similar case where the "magic number" for image scans is CVE > 8 but nobody has real clarity how they arrived at this number :-))

Reply:
the image scan number is/was set by ASSET - maybe log a separate ticket to update the scan display to say that ?